languages/elixir/exercises/concept/bitstrings/lib/dna.ex

```diff
@@ -0,0 +1,13 @@
+defmodule DNA do
+  def encode_nucleotide(codepoint), do: raise "Please implement encode_nucleotide/1"
```

Comment:
Just to be safe: the `do:` way of writing methods was covered in one of the prerequisite exercises, right?

Reply:
Good question, and yes:
https://github.com/exercism/v3/blob/c1ba3eb1bb6df1558e21c45a9afd43e3899be907/languages/elixir/exercises/concept/basics/.docs/after.md